spec/models/conditional_dependent_shorthand.rb

```diff
@@ -0,0 +1,8 @@
+class ConditionalDependentShorthand < ActiveRecord::Base
+  belongs_to :conditional_main
+  scope :condition, -> { where(condition: true) } 
+
+  counter_culture :conditional_main, if: :condition?, column_names: -> { {
```

Comment:
We'd need a test for `unless` and for the case of passing a `Proc` as well.

Reply:
Yep, and when passing an array of symbols for if / unless, and when using if and unless together.